Student/Student/Submissions/SubmissionRubric/RubricLongDescriptionViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+
+class RubricLongDescriptionViewController: UIViewController {
+
+    let longDescription: String
+    let navTitle: String
+
+    init(longDescription: String, title: String) {
+        self.longDescription = longDescription
+        self.navTitle = title
```

Comment:
can you just set `self.title = title` here?  Do you really need this `navTitle` variable?

Reply:
Oh I wasn't sure if that would work since the view hadn't loaded yet. But if that's not an issue, yeah, I will update this.